src/core/public/apm_system.ts

```diff
@@ -71,10 +73,16 @@ export class ApmSystem {
     start.application.currentAppId$.subscribe((appId) => {
       if (appId && this.apm) {
         this.closePageLoadTransaction();
-        this.apm.startTransaction(`/app/${appId}`, 'route-change', {
-          managed: true,
-          canReuse: true,
-        });
+        const apmTran = this.apm.startTransaction(
```

Comment:
For kibana-client, the goal is to have the data look like this in the longer term (at the moment we can't add the specific page navigated to):

| transaction.type  | transaction.name | labels.entityId
| ------------- | ------------- | ------------- | 
| change-route  | dashboard | list
| change-route  | dashboard | dashid1
| change-route  | dashboard | dashid2
| change-route  | apm | transaction1 
| change-route  | management | index-patterns